server/src/test/java/com/cloud/vm/UserVmManagerImplTest.java

```diff
@@ -276,4 +285,36 @@ private void configureValidateOrReplaceMacAddressTest(int times, String macAddre
         Mockito.verify(networkModel, Mockito.times(times)).getNextAvailableMacAddressInNetwork(Mockito.anyLong());
         Assert.assertEquals(expectedMacAddress, returnedMacAddress);
     }
+
+    @Test
+    public void validateRemoveTagsWhenExists() {
```

Comment:
You have test cases without any assert. How do you check if the method is doing what you want it to do?

Reply:
unfortunately is not easy to test.

Reply:
fixed